src/main/java/com/gregtechceu/gtceu/data/recipe/misc/MetaTileEntityLoader.java

```diff
@@ -363,6 +364,9 @@ public static void init(Consumer<FinishedRecipe> provider) {
         registerMachineRecipe(provider, GTMachines.ITEM_IMPORT_BUS, " C", " M", 'M', HULL, 'C', CustomTags.TAG_WOODEN_CHESTS);
         registerMachineRecipe(provider, GTMachines.ITEM_EXPORT_BUS, " M", " C", 'M', HULL, 'C', CustomTags.TAG_WOODEN_CHESTS);
 
+        registerMachineRecipe(provider, GTMachines.WORLD_ACCELERATOR, "FSF", "EHE", "FSF", 'F', FIELD_GENERATOR, 'S', SENSOR, 'E', EMITTER, 'H', HULL);
```

Comment:
you shouldn't add a recipe if `ConfigHolder.INSTANCE.machines.enableWorldAccelerators` is false.